include/z64camera.h

```diff
@@ -0,0 +1,389 @@
+    /* 0x38 */ CAM_SET_RFIXED1, // Various cutscenes, zooms into npcs face
+    /* 0x39 */ CAM_SET_TREASURE_CHEST_MINIGAME, // Treasure Chest Shop in East Clock Town, minigame location "TRESURE1"
+    /* 0x3A */ CAM_SET_HONEY_AND_DARLING_1, // Honey and Darling Minigames "BOMBBASKET"
+    /* 0x3B */ CAM_SET_CIRCLE8, // Used by Stone Tower moving platforms, Falling eggs in Marine Lab, Bugs into soilpatch cutscene
+    /* 0x3C */ CAM_SET_BIRDS_EYE_VIEW_1, // Camera is a top-down view. Used in Fishermans minigame and Deku Palace "FUKAN1"
+    /* 0x3D */ CAM_SET_DUNGEON3, // Generic dungeon camera 3, used in various places
+    /* 0x3E */ CAM_SET_TELESCOPE, // Observatory telescope and Curiosity Shop Peep-Hole "TELESCOPE"
+    /* 0x3F */ CAM_SET_ROOM0, // Certain rooms eg. inside the clock tower
+    /* 0x40 */ CAM_SET_RCIRC0, // Used by a few NPC cutscenes, focus close on the NPC
+    /* 0x41 */ CAM_SET_CIRCLE9, // Used by Sakon Hideout entrance and Deku Palace Maze
+    /* 0x42 */ CAM_SET_ONTHEPOLE, // Somewhere in Snowhead Temple and Woodfall Temple
+    /* 0x43 */ CAM_SET_INBUSH, // Various bush environments eg. grottos, Swamp Spider House, Termina Field grass bushes, Deku Palace near bean "INBUSH"
+    /* 0x44 */ CAM_SET_BOSS_MAJORA, // Majora's Lair: "BOSS_LAST" 
+    /* 0x45 */ CAM_SET_BOSS_TWINMOLD, // Twinmold's Lair: "BOSS_INI"
+    /* 0x46 */ CAM_SET_BOSS_GOHT, // Goht's Lair: "BOSS_HAK" 
+    /* 0x47 */ CAM_SET_BOSS_GYORG, // Gyorg's Lair: "BOSS_KON"
+    /* 0x48 */ CAM_SET_CONNECT0, // Smoothly and gradually return camera to Player after a cutscene
+    /* 0x49 */ CAM_SET_PINNACLE_ROCK, // Pinnacle Rock trench "MORAY"
+    /* 0x4A */ CAM_SET_NORMAL2, // Generic camera 2, used in various places
+    /* 0x4B */ CAM_SET_HONEY_AND_DARLING_2, // "BOMBBOWL"
+    /* 0x4C */ CAM_SET_CIRCLEA, // Unknown, Circle 10
+    /* 0x4D */ CAM_SET_WHIRLPOOL, // Great Bay Temple Central Room Whirlpool
+    /* 0x4E */ CAM_SET_CUCCO_SHACK, // "KOKKOGAME"
+    /* 0x4F */ CAM_SET_GIANT, // Giants Mask in Twinmold's Lair
+    /* 0x50 */ CAM_SET_SCENE0, // Entering doors to a new scene
+    /* 0x51 */ CAM_SET_ROOM1, // Certain rooms eg. some rooms in Stock Pot Inn
+    /* 0x52 */ CAM_SET_WATER2, // Swimming as Zora in Great Bay Temple
+    /* 0x53 */ CAM_SET_WOODFALL_SWAMP, // Woodfall inside the swamp, but not on the platforms, "SOKONASI"
+    /* 0x54 */ CAM_SET_FORCEKEEP, // Unknown
+    /* 0x55 */ CAM_SET_PARALLEL1, // Unknown
+    /* 0x56 */ CAM_SET_START1, // Used when entering the lens cave
+    /* 0x57 */ CAM_SET_ROOM2, // Certain rooms eg. Deku King's Chamber, Ocean Spider House "ROOM2"
+    /* 0x58 */ CAM_SET_NORMAL4, // Generic camera 4, used in Ikana Graveyard
+    /* 0x59 */ CAM_SET_ELEGY_SHELL, // cutscene after playing elegy of emptyness and spawning a shell "SHELL"
+    /* 0x5A */ CAM_SET_DUNGEON4, // Used in Pirates Fortress Interior, hidden room near hookshot
+    /* 0x5B */ CAM_SET_MAX
+} CameraSettingType;
```

Comment:
I think bearing in mind that the names will likely deviate further from the originals, it's a good idea to have them all somewhere, be it in the enum or in the file in which the data appears.

Reply:
I can just add the original name to each setting, regardless of whether it's identical or not to its current name